manifests/dotnet.yml

```diff
@@ -162,6 +173,7 @@ tests/:
         Test_Lfi_UrlQuery: v2.51.0
       test_libddwaf.py:
         Test_Libddwaf_Version: v3.4.1
+        Test_Libddwaf_Version_CmdI: missing_feature
```

Comment:
if we are going to have multiple libddwaf min versions, it will worth to move the test to each test file, like we are doing with `Rules_Version`

Reply:
Good point. I have updated the code addressing your comment.